### PR TITLE
Microsoft 365 Health/Announcements Connector (Independent Publisher)

### PR DESCRIPTION
**Pull request by techienickb**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

image
image
image
image

image

### CODE REVIEW

**Comment by natalie-pienkowska** [comment]
Thanks for submitting an Independent Publisher Connector! :) Since this is a connector to a 1st party service, we will be in touch with you shortly.

File: independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json
Comment (sriyen-msft) [inline independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json:166]: @techienickb Please do not add empty definitions in the swagger as they would not add any value to your connector.

File: independent-publisher-connectors/Microsoft 365 Service Health/apiProperties.json
Comment (sriyen-msft) [inline independent-publisher-connectors/Microsoft 365 Service Health/apiProperties.json:8]: @techienickb Please replace clientID value with "Dummy" value as it is not allowed to expose any security related information.

File: independent-publisher-connectors/Microsoft 365 Service Health/apiProperties.json
Comment (sriyen-msft) [inline independent-publisher-connectors/Microsoft 365 Service Health/apiProperties.json:48]: @techienickb Can you please add `stackOwner` property as it is required metadata property.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by techienickb** [comment]
@sriyen-msft updated as requested

File: independent-publisher-connectors/Microsoft 365 Service Health/apiProperties.json
Comment (sriyen-msft) [inline independent-publisher-connectors/Microsoft 365 Service Health/apiProperties.json:49]: @techienickb The stackOwner should be the owner of the service or product. Please follow the instructions mentioned in the [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-4-add-metadata).

File: independent-publisher-connectors/Microsoft 365 Service Health/apiProperties.json
Comment (sriyen-msft) [inline independent-publisher-connectors/Microsoft 365 Service Health/apiProperties.json:8]: @techienickb You can just provide dummy string as value, like below: 
`"clientId": "[[DUMMY]]"`

File: independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json
Comment (sriyen-msft) [inline independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json:9]: @techienickb Please provide `email` property as well.

File: independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json
Comment (sriyen-msft) [inline independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json:19]: @techienickb These category values are incorrect. Please provide the predefined category values only. And maximum two categories are allowed.

Documentation - [Product or end service metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-4b-product-or-end-service-metadata)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by techienickb** [comment]
Updated to use v1.0 of graph as this has been released and is no longer in beta

File: independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json
Comment (techienickb) [reply independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json]: Adjusted as requested, I've also switched from beta graph to v1.0 as the api is now released

File: independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json
Comment (sriyen-msft) [inline independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json:29]: @techienickb These empty operations have not removed, please remove them all.

File: independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json
Comment (sriyen-msft) [reply independent-publisher-connectors/Microsoft 365 Service Health/apiDefinition.swagger.json:73]: @techienickb Ping

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by techienickb** [comment]
@sriyen-msft will try again, I originally did the descriptions in the GUI and re-exported which is why some lower level object descriptions were missed

**Comment by Laskewitz** [comment]
@natalie-pienkowska just an FYI... this is a Graph API connector. Should probably go into the standard bucket. You might've already seen this - but just to be sure :)

**Comment by techienickb** [comment]
@sriyen-msft fixed swagger validation errors, should be good to go, unless MS wants to make it a first party connector (which would be nice)

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@techienickb Apologies for the delay in response, We are actively working on preparing and deploying your connector to publish environments. As you know that we are deploying the connector as a Standard connector, For that, we need to go thru a few guidelines and approvals from the concerned teams (ex. Power BI API team) which is a bit lengthy process. Please allow us some more time, we will keep you posted with the updates and deployment plan soon. Thank you so much for your patience.

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

**Comment by Amjed-Ayoub** [comment]
Hi @techienickb, I hope you are doing well.
Unfortunately, we couldn't get a permission from the Graph API due to an internal reason. We are looking for alternate method of authentication for your connector. We currently don’t have an estimated date in mind for deploying your connector. Rest assured that we are doing our best trying to get authorization for your connector.
Apologies for any inconvenience this may cause.
Thank you so much for your patience and understanding.

**Comment by vmanoharas** [comment]
Hello @techienickb, 
I hope you are doing well.
Apologies for the delay in response, as the connector certification team was in transition, we could not take your connector further. If you would like to go as independent publisher with your connector. Could you please close this and resubmit as new connector? so that we can adhere to new process. Since this is on-hold for many days.

**Comment by vmanoharas** [comment]
Hello @techienickb, 

This PR is open from more than three years, I will close this PR, kindly create a new PR.

**Pull request closed without merging**
